### PR TITLE
chore: update husky + unify publishing

### PR DESCRIPTION
**Pull request by blakebyrnes**

### CODE REVIEW

File: package.build.json [hunk 1]
```diff
@@ -10,24 +7,5 @@
...
-      "plugin-utils",
-      "plugins/*",
-      "puppet",
-      "puppet-chrome",
-      "mitm",
-      "mitm-socket",
-      "testing",
-      "../../commons/build"
-    ]
-  },
```
Comment (calebjclark) [inline package.build.json:29]: Shouldn't workspaces be left in the package.build.json? How else will it find the packages when `yarn build` locally?

Comment (blakebyrnes) [reply package.build.json:29]: I changed prepare-build and prepare-dist to automatically resolve the nested packages from the regular package.json

Comment (blakebyrnes) [reply package.build.json:29]: Sorry, backing up, this pr changes the root package.json, package.build.json, package.dist.json to behave like the nested projects - the .build.json and .dist.json files just provide the overrides

Comment (calebjclark) [reply package.build.json:29]: got it.

**Pull request merged**